pkg/operator/ceph/file/filesystem_test.go

```diff
@@ -131,6 +146,37 @@ func TestCreateFilesystem(t *testing.T) {
 	assert.ElementsMatch(t, []string{"rook-ceph-mds-myfs-a", "rook-ceph-mds-myfs-b"}, testopk8s.DeploymentNamesUpdated(deploymentsUpdated))
 	testopk8s.ClearDeploymentsUpdated(deploymentsUpdated)
 
+	// Increasing the number of data pools should be successful.
+	createdNewDataPool := false
+	associatedNewDataPool := false
+	executor = &exectest.MockExecutor{
+		MockExecuteCommandWithOutputFile: func(command string, outFileArg string, args ...string) (string, error) {
```

Comment:
This function should set `createdNewDataPool = false` and `associatedNewDataPool = false` at the beginning of each invocation to prevent false positives if create or associate is supposed to be called multiple times in future tests.